content/en/docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -1,184 +1,232 @@
+## Set up Kubernetes
 
-```shell
-systemctl mask firewalld.service
-systemctl stop firewalld.service
+1. Install kubernetes-kubeadm for your OS:
 
-systemctl disable iptables.service
-systemctl stop iptables.service
-```
+    Fedora Atomic Host
 
-**Configure the Kubernetes services on the master.**
+    <pre><tt># <b>rpm-ostree install kubernetes-kubeadm ethtool ebtables runc cri-tools -r</b>
+    <i>Log in again after automatic reboot</i>
+    </tt></pre>
 
-* Edit `/etc/kubernetes/apiserver` to appear as such.  The service-cluster-ip-range IP addresses must be an unused block of addresses, not used anywhere else.  They do not need to be routed or assigned to anything.
+    Fedora Workstation or Server
 
-```shell
-# The address on the local server to listen to.
-KUBE_API_ADDRESS="--address=0.0.0.0"
+    <pre><tt># <b>dnf install kubernetes-kubeadm ethtool ebtables runc cri-tools -y</b>
+    </tt></pre>
 
-# Comma separated list of nodes in the etcd cluster
-KUBE_ETCD_SERVERS="--etcd-servers=http://127.0.0.1:2379"
+2. Enable kubelet service, which manages pods on each node. (The kubelet service will start automatically after you run kubeadm init in the next step.)
 
-# Address range to use for services
-KUBE_SERVICE_ADDRESSES="--service-cluster-ip-range=10.254.0.0/16"
+    <pre><tt># <b>systemctl enable --now kubelet</b>
+    </tt></pre>
 
-# Add your own!
-KUBE_API_ARGS=""
-```
+3. Initialize Kubernetes. Options choose a set of IP addresses used for pods on the node and skips checks that will fail because of a later docker version:
 
-* Edit `/etc/etcd/etcd.conf` to let etcd listen on all available IPs instead of 127.0.0.1. If you have not done this, you might see an error such as "connection refused".
+    <pre><tt># <b>kubeadm init --pod-network-cidr=10.244.0.0/16 \
+        --ignore-preflight-errors=SystemVerification</b>
+    ...
+    Your Kubernetes master has initialized successfully!
+    </tt></pre>
 
-```shell
-ETCD_LISTEN_CLIENT_URLS="http://0.0.0.0:2379"
-```
+    If the initialization should fail, you can type **kubedm reset** and try the initialization again.
 
-* Start the appropriate services on master:
+    {{< note >}}
+    **Note:** The output from kubeadm init shows commands to run next. To add another node later, be sure to save the kubeadm join command line, which included token information you need to connect to the master. 
+    {{< /note >}}
 
-```shell
-for SERVICES in etcd kube-apiserver kube-controller-manager kube-scheduler; do
-    systemctl restart $SERVICES
-    systemctl enable $SERVICES
-    systemctl status $SERVICES
-done
-```
+4. Create a .kube directory in your home directory, add a configuration file, and change ownership (to be able to use kubectl later as a regular user, login as that user and append sudo to the cp and chown commands here):
```

Comment:
these instructions are part of the `kubeadm init` output, might be able to get away with mentioning that instead of copying commands here?